chatkit-core/src/main/kotlin/com/pusher/chatkit/users/api/UserApiTypeMapper.kt

```diff
@@ -0,0 +1,17 @@
+package com.pusher.chatkit.users.api
+
+import com.pusher.chatkit.users.state.UserInternalType
+import com.pusher.chatkit.util.DateApiTypeMapper
+
+internal class UserApiTypeMapper(private val dateApiTypeMapper: DateApiTypeMapper) {
+    fun toUserInternalType(user: UserApiType): UserInternalType {
+        return UserInternalType(
```

Comment:
```suggestion
    fun toUserInternalType(user: UserApiType) =
        UserInternalType(
```
is more legible as it removes the boiler-plate

Reply:
Suggestion would break syntax if committed as it though! (trailing `}` would be left on next line)

Reply:
yes of course, but not on the next lines but a few lines ahead! ;) the suggestion was for meant for clarity, the compiler would have immediatelly helped if that had been not clear ;) but yeah usually for not that trivial suggestions I check out the branch and make sure that they compile

Reply:
👍 have updated. 

FWIW i've stopped using GitHubs "commit suggestion" as it often causes subsequent compiler issues. I update the suggestion manually and then see what else complains :)